test/rpc/test_rpc_transactions.py

```diff
@@ -850,6 +850,7 @@ def test_add_deploy_account_transaction_on_incorrect_class_hash(deploy_account_d
         "starknet_addDeployAccountTransaction",
         params={"deploy_account_transaction": rpc_deploy_account_tx},
     )
+    print("DEBUG ex", ex)
```

Comment:
can be removed I guess

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet/pull/516/commits/46446d679838dc25f2014aa80f1ab4cdc96ec529